genealogy/geneweb/Portfile

```
@@ -1,109 +1,127 @@
-PortSystem 1.0
-
-name			geneweb
-version			5.02
-revision		3
-categories		genealogy
-maintainers		nomaintainer
-description		Sophisticated Genealogy System 
-long_description	GeneWeb is sophisticated genealogy software with a \
-			web interface.  It provides efficient relationship and consanguinity \
-			computing, support for very large database (i.e., hundreds of thousands \
-			of entries), and more.
-
-homepage		http://cristal.inria.fr/~ddr/GeneWeb
-master_sites		http://cristal.inria.fr/~ddr/GeneWeb/ftp/Src \
-			ftp://ftp.inria.fr/INRIA/Projects/cristal/${name}/Src
-distfiles		${name}-${version}.tgz
-checksums		md5 e8c6c789a9dd2f9155bf6eefa8786d01 \
-			sha1 e040496b3ce106cc7a508360cd0e4268bcb3e7b1 \
-			rmd160 fd284630e11957cf1265bfce9207e46d449a83ec
-platforms		darwin
-
-depends_build		port:ocaml \
-			port:camlp5
-
-patchfiles		patch-5.02-1
-
-use_parallel_build	no
-
-destroot.target		distrib
-
-set gwuser		${name}
-set gwgroup		${name}
-set logdir		${prefix}/var/log
-set langdir		${prefix}/share/${name}/gw
-set dbdir		${prefix}/var/db/${name}
-
-startupitem.create	yes
-startupitem.init        HOME=${prefix}/share/${name}
-startupitem.start	"su ${gwuser} -c \"cd ${prefix}/share/${name} && ./gw/gwd -bd${dbdir} -hd${langdir} -dd${langdir} -langen -log${logdir}/${name}.log -daemon\""
-startupitem.pidfile	none
-
-platform darwin {
-	if {${configure.build_arch} eq "x86_64" || ${configure.build_arch} eq "ppc64"} {
-		patchfiles-append patch-64-bit
-	}
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                geneweb
+version             7.1-beta
+revision            0
+github.setup        ${name} ${name} ${version} v
```

This is not how `github.setup` is intended to be used. The intended way is:
```suggestion
github.setup        geneweb geneweb 7.1-beta v
revision            0
```